samples/zigbee/light_bulb/src/main.c

```diff
@@ -515,6 +528,12 @@ void main(void)
 	bulb_clusters_attr_init();
 	level_control_set_value(dev_ctx.level_control_attr.current_level);
 
+        /** Initialize ZCL scene table */
+        zcl_scenes_init();
+
+	/* Settings should be loaded after zcl_scenes_init */
+	settings_load();
```

Comment:
Should we check returned value?

Reply:
added LOG_ERR if returned value is non-zero